apps/meteor/server/startup/migrations/v282.ts

```diff
@@ -0,0 +1,18 @@
+import { Statistics } from '@rocket.chat/models';
+
+import { getMatrixFederationStatistics } from '../../../app/federation-v2/server/infrastructure/rocket-chat/statistics';
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 282,
+	async up() {
+		await Statistics.updateMany(
+			{},
+			{
+				$set: {
+					matrixFederation: await getMatrixFederationStatistics(),
+				},
+			},
+		);
+	},
+});
```

Comment:
We strongly recommend avoid migrations, if something happens (during the release candidates or any other deployment) we cannot only revert to the old version.

Sometimes its not possible to continue without migrations, but I'm afraid this is not the case. 
Looks 100% useless to me

@sampaiodiego fyi

Reply:
Makes sense, honestly, it was to replace one value with another, but since I've removed the `$unset` in the previous change requested, it became completely useless now. 👍